chisel.lua

```diff
@@ -47,6 +47,31 @@ local default_material = {
 			{"default:silver_sandstone_brick","default_silver_sandstone_brick", "Silver Sandstonebrick"},
 			}
 
+
+
+if minetest.get_modpath( "bakedclay") then
+   local clay = {
+      {"white", "White"},
+      {"grey", "Grey"},
+      {"black", "Black"},
+      {"red", "Red"},
+      {"yellow", "Yellow"},
+      {"green", "Green"},
+      {"cyan", "Cyan"},
+      {"blue", "Blue"},
+      {"magenta", "Magenta"},
+      {"orange", "Orange"},
+      {"violet", "Violet"},
+      {"brown", "Brown"},
+      {"pink", "Pink"},
+      {"dark_grey", "Dark Grey"},
+      {"dark_green", "Dark Green"},
+   }
+
+   for _, clay in pairs(clay) do
+      table.insert(default_material,{"bakedclay:"..clay[1] , "baked_clay_" .. clay[1], clay[2] .. " Baked Clay"})
+   end
```

Comment:
A majority of the lines are indented using tabs. Please use tabs here as well to not make it an even larger mess.

Reply:
per default "kate" editor uses spaces as identation. Do not ask me why. I had to search in the prefs to find that. I hope I will find some time this weekend